### PR TITLE
Code Cleanup

### PR DESCRIPTION
**Pull request by clowenhg**
Several code cleanup changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,16 +28,24 @@
     "assetmanager": "^1.2.2",
     "complex-list": "^0.1.4",
     "config": "^1.21.0",
+    "connect-mongo": "^0.4.1",
+    "cookie-parser": "^1.1.0",
     "errorhandler": "^1.4.3",
+    "express": "^4.2.0",
     "express-jwt": "^5.0.0",
+    "express-session": "^1.1.0",
```
Comment (rjVapes) [inline package.json]: Is express-session currently used?

Comment (timelf123) [inline package.json]: not that I'm aware of. I think it is a remnant of "the land before JWT"

**Comment by adskjohn** [comment]
Any reason behind deleting test/lib/mean_test.js vs. making it work?  (assuming it doesn't)

**Comment by clowenhg** [comment]
It only really had two tests in it. One for aggregation which is gone, and one for menus which has been vastly improved in the actual menus test.

**Comment by adskjohn** [comment]
Cool, just wanted to make sure we're gaining or stay the same on coverage.  The code looks good to me, so if you've locally tested and it all runs fine, I'm good to merge it :)

**Pull request merged**